ruby/helpers/disclaimers.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+require "cli/ui"
+require_relative("colorizer")
+
+def billing
+  puts "\e[H\e[2J"
+  puts "========================== BILLING DISCLAIMER =============================".yellow
+  puts "Please note that running this code will create actual resources that may"
```

Comment:
Update to first sentence: 
Running this code creates actual resources that might incur charges.